data/assets/examples/renderable/renderablemodel/model_animation.asset

```diff
@@ -0,0 +1,58 @@
+-- Animation
+-- This example loads a model with a simple keyframe animation. The animation starts at
+-- a set time, in this case "2024 07 09 12:00:00". The parent is not set which defaults
+-- the position of the model to the center of the Solar System.
+
+-- Load the asset of the Sun to illuminate the model
+local sun = asset.require("scene/solarsystem/sun/transforms")
```

Comment:
If we don't need the lighting for this, it might be better to not have it so that the example becomes a bit simpler